src/security/kuzzleProfile.js

```diff
@@ -1,7 +1,7 @@
 var
   KuzzleSecurityDocument = require('./kuzzleSecurityDocument');
 
-function KuzzleProfile(kuzzleSecurity, id, content) {
+function Profile(kuzzleSecurity, id, content) {
```

Comment:
`kuzzleSecurity` here is a variable name. I think that variable names are out of the scope of this refactoring.